client.go

```diff
@@ -88,6 +95,14 @@ var (
 	// quote call as the miner fee if the fee estimation in lnd's wallet
 	// failed because of insufficient funds.
 	MinerFeeEstimationFailed btcutil.Amount = -1
+
+	// LndRequiredActiveSubservers is a list of subservers that need to be
+	// activated in lnd for loop to work properly. These are still
+	// experimental and are only activated through a build flag. Therefore
+	// they might not be present if the user manually compiled.
+	LndRequiredActiveSubservers = []string{
+		"signrpc", "walletrpc", "chainrpc", "invoicesrpc",
```

Comment:
Maybe string constants? Probably more necessary if the version checking code is moved into a shared module like `lndclient`.